demo/firebase-storage-multiupload-demo.html

```diff
@@ -0,0 +1,201 @@
+<!--
+@license
+Copyright 2016 Google Inc. All Rights Reserved.
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file or at
+https://github.com/firebase/polymerfire/blob/master/LICENSE
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../paper-progress/paper-progress.html">
+<link rel="import" href="../../paper-toast/paper-toast.html">
+<link rel="import" href="../polymerfire.html">
+
+<dom-module id="firebase-storage-multiupload-demo">
+  <template>
+    <firebase-app
+      auth-domain="polymerfire-test-6abc0.firebaseapp.com"
+      database-url="https://polymerfire-test-6abc0.firebaseio.com"
+      api-key="AIzaSyBxZI1BAgX9obocHQw_yhmHV9BIOpaNWZo"
+      storage-bucket="polymerfire-test-6abc0.appspot.com"></firebase-app>
+    <firebase-auth id="auth" user="{{user}}" provider="google" on-error="catchError"></firebase-auth>
+
+    <button on-tap="signIn">Sign In</button><br/>
+
+    <template is="dom-if" if="{{user}}">
+      <firebase-storage-multiupload
+        id="fs"
+        path="/users/{{user.uid}}/files"
+        files="[[files]]"
+        upload-tasks="{{uploadedFiles}}"
+        on-error="catchError"
+        force-unique
+        auto>
+      </firebase-storage-multiupload>
+
+      <input id="file-uploader" type="file" on-change="onFileUpload" multiple /> <br/>
+      <hr/>
+      <template is="dom-repeat" items="[[uploadedFiles]]">
+        <div style="padding: 20px">
+          <firebase-storage-upload-task
+              task="[[item]]"
+              id="task-[[index]]"
+              bytes-transferred="{{item.bytesTransferred}}"
+              total-bytes="{{item.totalBytes}}"
+              state="{{item.state}}"
+              download-url="{{item.downloadUrl}}"
+              metadata="{{item.metadata}}"
+              path="{{item.path}}"></firebase-storage-upload-task>
+
+
+
+          path from snapshot: {{item.snapshot.ref.fullPath}} <br/>
```

Comment:
What about changing two-way data flow to one-way (double curly brackets — {{ }} to double square brackets — [[ ]]) around the all code where it is needed?